doc/classes/CheckBox.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		A checkbox allows the user to make a binary choice (choosing only one of two possible options). It's similar to [CheckButton] in functionality, but it has a different appearance. To follow established UX patterns, it's recommended to use CheckBox when toggling it has [b]no[/b] immediate effect on something. For example, it could be used when toggling it will only do something once a confirmation button is pressed.
 		See also [BaseButton] which contains common properties and methods associated with this node.
+		[b]Note:[/b] The [CheckBox] will be configured as a radio button if a [ButtonGroup] is assigned to it. See [member BaseButton.button_group] and [ButtonGroup].
```

Comment:
This is true for any button. I mean, we can repeat it for CheckBox specifically, I guess, but this seems to be missing from the docs on Button/BaseButton too.

Reply:
It seems that only the `CheckBox` will have a different appearance when it is configured as a radio button.